store/src/java/com/zimbra/cs/service/admin/FlushCache.java

```diff
@@ -328,4 +328,16 @@ private static String sanitizeImapCacheTypes(String cacheTypes) {
         }
         return imapTypes.isEmpty() ? null : Joiner.on(",").join(imapTypes);
     }
+
+    /**
+     * Flushes config level cache of all other server
+     * @param zimbraSoapContext
+     * @return nothing
+     * @throws ServiceException
+     */
+    public static void flushConfigCacheOnAllServers(ZimbraSoapContext zimbraSoapContext) throws ServiceException {
```

Comment:
not sure if we should start adding different methods based on type of cache which needs to be flushed
any caller can do this to flush cache on all servers
```
                CacheSelector cacheSel = new CacheSelector(true /* allServers */, CacheEntryType.config.toString());
                FlushCacheRequest fcReq = new FlushCacheRequest(cacheSel);
                try {
                    FlushCache.doFlushCache(this, context, fcReq);
                } catch (ServiceException se) {
                    // Error log
                }
```

Reply:
if you check my code then it's restricting to config cache only :D

Reply:
Got it. Swati will make changes accordingly. Thanks